### PR TITLE
CT-2901 | Billing info UI tests & refactor

### PR DESCRIPTION
**Pull request by eugenezhernakov**
* Add billing info tests to cover input validation, postcode (UK, CA and US), payment with billing info
* Refactor `TestData` and `Selectors` to better organise all variables
* Rename some tests to shorten and simplify

### CODE REVIEW

File: Examples/ObjectiveCExampleApp/ObjectiveCExampleAppUITests/Additions/XCUIApplication+Additions.swift [hunk 1]
```diff
@@ -24,94 +24,161 @@ extension XCUIApplication {
...
+            return buttonWithIdentifier(Selectors.BillingInfo.addAddressLineButton)
+        }
+    }
+    
+    var fieldErrorLabel: String? {
+        get {
+            guard let labelElement = textWithIdentifier(Selectors.BillingInfo.fieldErrorLabel) else {
+                return nil
+            }
+            return labelElement.label
```
Comment (RafalOzogJudopay) [inline Examples/ObjectiveCExampleApp/ObjectiveCExampleAppUITests/Additions/XCUIApplication+Additions.swift:181]: I wonder whether it makes sense to use guard in this case? If textWithIdentifier may return nullable result, maybe it's redundant. @stefan-tudor could you say your opinion, please?

File: Examples/ObjectiveCExampleApp/ObjectiveCExampleAppUITests/CardPayment/CardPaymentUITests.swift [hunk 2]
```diff
@@ -78,166 +78,166 @@ final class CardPaymentUITests: XCTestCase {
...
         app.tokenPaymentButton?.tap()
-        app.securityCodeTextField?.tapAndTypeText(TestData.CARD_SECURITY_CODE)
+        app.securityCodeTextField?.tapAndTypeText(TestData.CardDetails.CARD_SECURITY_CODE)
         app.cardDetailsSubmitButton?.tap()
         tapCompleteButton(app)
         assertResultObject(app, "Payment", "AuthCode: ", "Success")
     }
     
-    func testSuccessfulTokenPreauthReceiptObjectContainsRelevantInfo() {
+    func testSuccessfulTokenPreauth() {
```
Comment (RafalOzogJudopay) [inline Examples/ObjectiveCExampleApp/ObjectiveCExampleAppUITests/CardPayment/CardPaymentUITests.swift:145]: Nice, I agree these shortened test names are easier for brain to understand.

**Review by RafalOzogJudopay — APPROVED** [review]
Looks good, I left some comments below (above), but no blockers detected.

Please ensure that your branch is updated with 'master' after our recent updates. I wonder wether some merge conflicts may need to be resolved (in Ravelin tests in particular).

Also, let's get @stefan-tudor confirmation before merging (just in case, so we don't mess with iOS release stuff he's working on, by merging in a wrong moment).

**Review by stefan-tudor — APPROVED** [review]

File: Examples/ObjectiveCExampleApp/ObjectiveCExampleAppUITests/Additions/XCUIApplication+Additions.swift (hunk 1 above)
Comment (eugenezhernakov) [reply Examples/ObjectiveCExampleApp/ObjectiveCExampleAppUITests/Additions/XCUIApplication+Additions.swift]: Changed, thanks both 💯

**Pull request merged**